pkg/collector/corechecks/containerlifecycle/telemetry.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package containerlifecycle
+
+import "github.com/DataDog/datadog-agent/pkg/telemetry"
+
+const (
+	PodKind       = "pod"
+	ContainerKind = "container"
+)
+
+var emittedEvents = telemetry.NewCounterWithOpts(
+	checkName,
+	"emitted_events",
+	[]string{"object_kind"}, // "object_kind" can be PodKind or ContainerKind
```

Comment:
can we have also the type of event? 
`stop` `restart`?

Reply:
The container_lifecycle check doesn't send that information. Here's what it sends: https://github.com/DataDog/datadog-agent/blob/a694f3a34853231d9f791669ea763cec874a2144/pkg/collector/corechecks/containerlifecycle/processor.go#L71-L104

Notice that "event type" is always "delete".

But I can add the exit_code for containers.

Reply:
I think you can add event type event it is always delete. it can change in the futur.

for the exit_code I think it can increase the cardinality for no good reason.

Reply:
Sounds good. I added the event type :+1: